cmd/crank/install.go

```diff
@@ -69,11 +69,6 @@ type installCmd struct {
 	Provider      installProviderCmd `cmd:"" help:"Install a Provider package."`
 }
 
-// Run runs the install cmd.
-func (c *installCmd) Run(_ *buildChild) error {
-	return nil
-}
```

Comment:
Was this intentional? I see you said you were leaving install for now - I'm guessing this is a no-op change?

Reply:
Yes, the actual commands are implemented in the subcommands' Run. Not really sure why these were there.